bazel-integration-testing.patch

```diff
@@ -0,0 +1,54 @@
+diff tools/repositories.bzl tools/repositories.bzl
+index 8364dac..42e998d 100644
+--- tools/repositories.bzl
++++ tools/repositories.bzl
+@@ -15,11 +15,6 @@
+ load("//tools:bazel_hash_dict.bzl", "BAZEL_HASH_DICT")
+ load(":common.bzl", "BAZEL_VERSIONS")
+ 
+-_BAZEL_BINARY_PACKAGES = [
+-  "http://releases.bazel.build/{version}/release/bazel-{version}{installer}-{platform}.{extension}",
+-  "https://github.com/bazelbuild/bazel/releases/download/{version}/bazel-{version}{installer}-{platform}.{extension}",
+-]
+-
+ def _get_platform_name(rctx):
+   os_name = rctx.os.name.lower()
+ 
+@@ -45,7 +40,36 @@ def _get_installer(rctx):
+     extension = "sh"
+     installer = "-installer"
+ 
+-  urls = [url.format(version=version, installer=installer, platform=platform, extension=extension) for url in _BAZEL_BINARY_PACKAGES]
++  filename = "bazel-{version}{installer}-{platform}.{extension}".format(
```

Comment:
😎 Should make a note to upstream

Reply:
I sent the PR already, sorry should have included PR number in the patch filename but I did them in the wrong order